Data/ByteString/Short/Internal.hs

```diff
@@ -336,20 +301,14 @@ instance Read ShortByteString where
 -- | @since 0.10.12.0
 instance GHC.Exts.IsList ShortByteString where
```

Comment:
Is there any reason not to newtype-derive this instance as well?

Reply:
Oh, right. We need to define `Item`. But please also include an explicit definition for `fromListN`.